bogo/src/main.rs

```diff
@@ -1600,6 +1606,14 @@ pub fn main() {
             "-server-preference" => {
                 opts.server_preference = true;
             }
+            "-wait-for-debugger" => {
+                #[cfg(windows)] {
```

Comment:
This formatting looks surprising to me, is rustfmt skipping this? I think the usual formatting would be like

```rust
#[cfg(windows)]
{
    panic(..);
}
```

Reply:
> is rustfmt skipping this?

oh bother, I think it must be. I'll take a look.

Reply:
Hmmmm I can't get a local `cargo fmt` to complain, but it does seem to be looking at `bogo/src/main.rs`:

```
[daniel@noire:~/Code/Rust/rustls/bogo]$ git rev-parse HEAD
edd6a68c50d6e65bf37ee71884bcf17619d4e6ed

[daniel@noire:~/Code/Rust/rustls/bogo]$ cargo fmt --all --verbose --check | grep bogo
[bin (2021)] "/home/daniel/Code/Rust/rustls/bogo/src/main.rs"
[test (2021)] "/home/daniel/Code/Rust/rustls/rustls/tests/bogo.rs"
rustfmt --edition 2021 --check /home/daniel/Code/Rust/rustls/bogo/src/main.rs <snipped>
```

Reply:
Fixed manually for now.

Reply:
Weird -- it doesn't seem to be in a macro context, either.

Reply:
I think maybe rustfmt gives up if it can't find a way to format some code to fit in the line width? And this code is pretty tall and wide.